android/shopify_buy_plugin/src/main/java/com/shopify/unity/buy/UnityAndroidPayFragment.java

```diff
@@ -39,9 +80,68 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
             unityDelegateObjectName = bundle.getString(EXTRA_UNITY_DELEGATE_OBJECT_NAME);
             cart = bundle.getParcelable(EXTRA_PAY_CART);
             countryCode = bundle.getString(EXTRA_COUNTRY_CODE);
+            androidPayEnvironment = bundle.getInt(EXTRA_ANDROID_PAY_ENVIRONMENT,
+                    WalletConstants.ENVIRONMENT_TEST);
+            androidPublicKey = bundle.getString(EXTRA_PUBLIC_KEY);
         }
 
         // Don't recreate this fragment during configuration change.
         setRetainInstance(true);
+
+        googleApiClient = new GoogleApiClient.Builder(this.getActivity())
+                .addApi(Wallet.API, new Wallet.WalletOptions.Builder()
+                        .setEnvironment(androidPayEnvironment)
+                        .setTheme(WalletConstants.THEME_DARK)
```

Comment:
Coded to be dark theme for now. If developers ask to skin this in the future we can make that change then.